packages/template-retail-react-app/app/pages/store-locator/index.jsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import React, {useState} from 'react'
+import {useIntl} from 'react-intl'
+import {Box, Container} from '@salesforce/retail-react-app/app/components/shared/ui'
+import Seo from '@salesforce/retail-react-app/app/components/seo'
+import StoreLocatorContent from '@salesforce/retail-react-app/app/components/store-locator-modal/store-locator-content'
+import {useSearchStores} from '@salesforce/commerce-sdk-react'
+import {
+    DEFAULT_STORE_LOCATOR_COUNTRY_CODE,
+    DEFAULT_STORE_LOCATOR_POSTAL_CODE,
+    STORE_LOCATOR_DISTANCE
+} from '@salesforce/retail-react-app/app/constants'
+import {useForm} from 'react-hook-form'
+
+const StoreLocator = () => {
+    const intl = useIntl()
+
+    const [searchStoresParams, setSearchStoresParams] = useState({
+        countryCode: DEFAULT_STORE_LOCATOR_COUNTRY_CODE,
+        postalCode: DEFAULT_STORE_LOCATOR_POSTAL_CODE
+    })
+    const form = useForm({
+        mode: 'onChange',
+        reValidateMode: 'onChange',
+        defaultValues: {
+            countryCode: searchStoresParams?.countryCode,
+            postalCode: searchStoresParams?.postalCode
+        }
+    })
+    var searchStoresData = useSearchStores({
+        parameters: {
+            countryCode: searchStoresParams.countryCode,
+            postalCode: searchStoresParams.postalCode,
+            locale: intl.locale,
+            maxDistance: STORE_LOCATOR_DISTANCE
+        }
+    })
+    var storesInfo = []
+    if (searchStoresData.data !== undefined && searchStoresData.data.data !== undefined)
+        storesInfo = searchStoresData.data.data
```

Comment:
I think we can follow what's done in the other file: https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1827/files#r1646765524